Sources/SynchronizationManager.swift

```diff
@@ -216,16 +227,19 @@ public class SynchronizationManager: PersistenceIntegration {
 
     // Returns regular (non-relationship) field to property mappings.
     internal func propertyMapping(for entryType: EntryPersistable.Type, and fields: [FieldName: Any]) -> [FieldName: String] {
+        // Filter out user-defined properties that represent relationships.
+        guard let persistentRelationshipPropertyNames = try? persistentStore.relationships(for: entryType) else {
+            assertionFailure("Could not filter out user-defined properties that represent relationships.")
```

Comment:
I am not 100% sure if this is really necessary to filter out the items in order to have a stable running app.
@loudmouth what do you think?